server/community.go

```diff
@@ -32,3 +33,18 @@ func (s *Server) addHacktoberfestLabel(ctx context.Context, pr *model.PullReques
 		return
 	}
 }
+
+func (s *Server) postPRWelcomeMessage(ctx context.Context, pr *model.PullRequest) {
+	// Only post welcome Message for community member
+	if s.IsOrgMember(pr.Username) {
+		return
+	}
+
+	msg := strings.ReplaceAll(s.Config.PRWelcomeMessage, "USERNAME", "@"+pr.Username)
+
+	err := s.sendGitHubComment(ctx, pr.RepoOwner, pr.RepoName, pr.Number, msg)
+	if err != nil {
+		mlog.Warn("Error while commenting PR welcome message", mlog.Err(err))
+		return
+	}
```

Comment:
Please propagate this error upwards and handle it at the call site.

Reply:
Done